### ISSUE TITLE
Modif: Gal/Col Obras. Insertar más de 4 imágenes

### ISSUE THREAD

**Issue by ElisaArteGlobal** [emamnuelmesa/Arte-Global#553]
![Captura de pantalla 2019-02-05 a las 12.45.07.png](https://images.zenhubusercontent.com/5a098a0d8a75884b90872ae7/94a1257d-3fb2-403d-bfd3-91bc380e6266)
¿Se puede hacer un desarrollo para incluir más de 4 imágenes? ¿cuántas se podrían

**Comment by miguelGonzalezQuasar** [comment]
Sí, se pueden incluir mas imágenes. ¿Cuántas son necesarias añadir?

Un saludo,
